nekoyume/Assets/_Scripts/L10n/L10nTextMeshProUGUI.cs

```diff
@@ -123,37 +124,47 @@ private void Awake()
 
             if (L10nManager.CurrentState == L10nManager.State.Initialized)
             {
+                SetLanguage();
                 SetLanguageTypeSettings(L10nManager.CurrentLanguageTypeSettings);
                 SubscribeLanguageChange();
             }
             else
             {
                 L10nManager.OnInitialize.Subscribe(_ =>
                 {
+                    SetLanguage();
                     SetLanguageTypeSettings(L10nManager.CurrentLanguageTypeSettings);
                     SubscribeLanguageChange();
                 }).AddTo(gameObject);
             }
-
-            if (!string.IsNullOrEmpty(l10nKey))
-            {
-                Text.text = L10nManager.Localize(l10nKey);
-            }
         }
 
         private void OnDestroy()
         {
+            _l10nManagerOnLanguageChangeDisposable?.Dispose();
+            _l10nManagerOnLanguageChangeDisposable = null;
             _l10nManagerOnLanguageTypeSettingsChangeDisposable?.Dispose();
             _l10nManagerOnLanguageTypeSettingsChangeDisposable = null;
         }
 
         private void SubscribeLanguageChange()
         {
+            _l10nManagerOnLanguageChangeDisposable?.Dispose();
+            _l10nManagerOnLanguageChangeDisposable = 
+                L10nManager.OnLanguageChange.Subscribe(_ => SetLanguage());
             _l10nManagerOnLanguageTypeSettingsChangeDisposable?.Dispose();
             _l10nManagerOnLanguageTypeSettingsChangeDisposable =
                 L10nManager.OnLanguageTypeSettingsChange.Subscribe(SetLanguageTypeSettings);
         }
 
+        private void SetLanguage()
+        {
+            if (!string.IsNullOrEmpty(l10nKey))
```

Comment:
```suggestion
            if (!string.IsNullOrWhiteSpace(l10nKey))
```

I think, it would be the developer's mistake if there was a white space. Why don't we check this as well?

Reply:
Thank you for recommend! I will commit the suggestion.